requests_toolbelt/utils/deprecated.py

```diff
@@ -4,6 +4,17 @@
 
 from requests import utils
 
+from .._compat import PY3 as _PY3
+
+_re_charset = r'<meta.*?charset=["\']*(.+?)["\'>]'
```

Comment:
You can actually write each of these as

```
br'...'
```

We're always expecting bytes and `r'...' == br'...'` on Python 2 and `r'...'.encode('ascii') == br'...'` on Python 3. =)